debugview/src/debug/kotlin/com/revenuecat/purchases/debugview/settings/SettingOffering.kt

```diff
@@ -0,0 +1,297 @@
+            ) {
+                Text(text = "Buy package", style = MaterialTheme.typography.subtitle2)
+            }
+            Button(
+                modifier = Modifier.padding(horizontal = 16.dp),
+                onClick = { screenViewModel.purchaseProduct(activity, rcPackage.product) },
+            ) {
+                Text(text = "Buy product", style = MaterialTheme.typography.subtitle2)
+            }
+        }
+        if (isSubscription) {
+            Divider()
+            Text(
+                modifier = Modifier.padding(16.dp),
+                text = "Subscription Options",
+                style = MaterialTheme.typography.subtitle2,
+            )
+            rcPackage.product.subscriptionOptions?.forEach { subscriptionOption ->
+                SettingSubscriptionOption(
+                    activity = activity,
+                    screenViewModel = screenViewModel,
+                    subscriptionOption = subscriptionOption,
+                    isDefaultOption = subscriptionOption == rcPackage.product.defaultOption,
+                )
+                if (subscriptionOption != rcPackage.product.subscriptionOptions?.last()) {
+                    Divider()
+                }
+            }
+        }
+    }
+}
+
+@Composable
+internal fun SettingSubscriptionOption(
+    activity: Activity,
+    screenViewModel: DebugRevenueCatViewModel,
+    subscriptionOption: SubscriptionOption,
+    isDefaultOption: Boolean,
+) {
+    Column(
+        modifier = Modifier
+            .fillMaxWidth()
+            .padding(horizontal = 16.dp),
+        horizontalAlignment = Alignment.End,
+    ) {
+        Row(modifier = Modifier.fillMaxWidth(), horizontalArrangement = Arrangement.SpaceBetween) {
+            Text(text = "Subscription Option ID:", style = MaterialTheme.typography.body1)
+            Text(text = subscriptionOption.id, style = MaterialTheme.typography.body2)
+        }
+        Row(modifier = Modifier.fillMaxWidth(), horizontalArrangement = Arrangement.SpaceBetween) {
+            Text(text = "Tags:", style = MaterialTheme.typography.body1)
+            Text(
+                text = subscriptionOption.tags.takeIf { it.isNotEmpty() }?.toString() ?: "None",
+                style = MaterialTheme.typography.body2,
+            )
+        }
+        Row(modifier = Modifier.fillMaxWidth(), horizontalArrangement = Arrangement.SpaceBetween) {
+            Text(text = "Pricing phases:", style = MaterialTheme.typography.body1)
+            val pricingPhasesString = subscriptionOption.pricingPhases.joinToString("\n") {
+                "- ${it.price.formatted}/${it.billingPeriod.iso8601}"
+            }
+            Text(
+                text = pricingPhasesString,
+                style = MaterialTheme.typography.body2,
+            )
+        }
+        Button(onClick = { screenViewModel.purchaseSubscriptionOption(activity, subscriptionOption) }) {
```

Comment:
Honestly, I'm not sure if this is too many options for a public debug view... But I thought it could be useful to visualize the different subscription options.